qiskit/visualization/dag_visualization.py

```diff
@@ -25,7 +25,7 @@
 
 
 @_optionals.HAS_GRAPHVIZ.require_in_call
-def dag_drawer(dag, scale=0.7, filename=None, style="color"):
+def dag_drawer(dag, scale=0.7, filename=None, style="color", show_node_id=False):
```

Comment:
Similarly here - it seems like an unrelated change.

Reply:
removed in ff987f1c6